Cargo.toml

```diff
@@ -46,6 +45,10 @@ postgres-types = { version = "0.2.4", features = ["derive"] }
 cron = { version = "0.15.0" }
 bytes = "1.1.0"
 structopt = "0.3.26"
+hmac = "0.12.1"
+sha1 = "0.10.6"
+digest = "0.10.7"
```

Comment:
I think that this dependency is unused?
```suggestion

```

Reply:
I had the same reflection and I think it's used as a dependency of `hmac`. It's probably here to set it's version transitively.

Reply:
Ah yes it's unused, initially I was trying to use it for https://docs.rs/digest/latest/digest/struct.CtOutput.html#method.ct_eq before I realised it's from the `subtle` crate